apps/web/lib/api/scrape-creators/get-social-content.ts

```
@@ -0,0 +1,213 @@
+  url.hostname = url.hostname.toLowerCase();
+
+  // Remove tracking params
+  [
+    "utm_source",
+    "utm_medium",
+    "utm_campaign",
+    "si",
+    "feature",
+    "igshid",
+    "t",
+  ].forEach((p) => url.searchParams.delete(p));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)